<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 127);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 italic 15pt &quot;Segoe UI&quot;;
color: rgb(255, 85, 0);</string>
   </property>
   <property name="text">
    <string>Type the colour of the words , and not the word text!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_check">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>190</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 127);
font: 700 italic 20pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>check</string>
   </property>
  </widget>
  <widget class="QLabel" name="text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>521</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 italic 20pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>text</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);
font: 700 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>score :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_score">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>50</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 18pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 15pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);
font: 700 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>time :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
